server-api/src/main/java/org/sonarsource/sonarlint/core/serverapi/rules/ServerRule.java

```diff
@@ -19,13 +19,21 @@
  */
 package org.sonarsource.sonarlint.core.serverapi.rules;
 
+import org.sonarsource.sonarlint.core.commons.Language;
+
 public class ServerRule {
   private final String name;
   private final String htmlDesc;
   private final String htmlNote;
+  private final String severity;
+  private String type;
```

Comment:
Why not final?

Reply:
Because I forgot to tell IDEA to generate it that way 😉 good catch!